### PR TITLE
Analyze and format given code for syntax errors

### PR DESCRIPTION
**Pull request by vishnusandeep969**
TS027ASL Linting - Analyze and format gave code for syntax errors

### CODE REVIEW

File: cssvalidation.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                        <ul>
+                        <li>Learn More . The W3C has a CSS Validator available too, for this purpose.</li>
+                        <li>for more visit :- </li>
+                        <li><a href="https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS" target="_blank">https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS</a></li>
+                        </ul> 
+                    </section>
+                    <section  data-transition="slide" data-background="#006400"  data-background-transition="zoom">
+                        <h4>task</h4>
+                        <ul>
+                            <li>Validate the CSS file in Troubleshooting Reposittory</li>
```
Comment (wandercode) [inline cssvalidation.html]: please give the proper linking here.

File: cssvalidation.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                    </section>
+                    <section data-transition="slide" data-background="#006400"  data-background-transition="zoom">
+                        <ul>
+                            <li>CSS has a similar story — you need to check that your property names are spelled correctly, property values are spelled correctly and are valid for the properties they are used on.</li>
+                            <img src="image/pic2.png">   
+                        </ul> 
+                    </section>
+                    <section data-transition="slide" data-background="#006400"  data-background-transition="zoom">
+                        <ul>
+                        <li>Learn More . The W3C has a CSS Validator available too, for this purpose.</li>
```
Comment (wandercode) [inline cssvalidation.html]: please give the proper linking here.

File: syntaxerrors.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                 <section  data-transition="slide" data-background="#008080" data-background-transition="zoom">
+                    <ul>
+                        <li>Linters</li>
+                        <li>Another good option to choose is a so-called Linter application, which not only points out errors, but can also flag up warnings about bad practices in your CSS, and other points besides. Linters can generally be customized to be stricter or more relaxed in their error/warning reporting</li>
+                   </ul>
+                    </section>
+                   <section  data-transition="slide" data-background="#008080" data-background-transition="zoom">
+                        <ul><li>There are many online linter applications, the best of which are probably,</li>
+                           <ul>
+                               <li>Dirty Markup (HTML, CSS, JavaScript), and</li>
```
Comment (wandercode) [inline syntaxerrors.html]: please give proper linking

File: syntaxerrors.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                               <li>CSS Lint (CSS only).</li>
```
Comment (wandercode) [inline syntaxerrors.html]: please give proper linking

File: syntaxerrors.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                </section>
+                <section  data-transition="slide" data-background="#008080" data-background-transition="zoom">
+                <h4>tasks</h4>
+                <ul>
+                    <li>Learn About how to Use Linters in Code Editors</li>
+                    <li><a href = "https://www.youtube.com/watch?v=cMrDePs86Uo">Learn how to use linters in VS code</a> (You can Use this Repo.<a href = "https://github.com/brainchildservices/Troubleshooting">Troubleshooting Reposittory).</a></li>
+                </ul>
+                </section>
+                <section  data-transition="slide" data-background="#008080" data-background-transition="zoom">
+                    <p>Ref Link:<a href = "https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS">https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/HTML_and_CSS</a> </p>
```
Comment (wandercode) [inline syntaxerrors.html]: remove this section

**Review by wandercode — CHANGES_REQUESTED** [review]

**Comment by vishnusandeep969** [comment]
requested changes done

**Comment by vishnusandeep969** [comment]
all the links to open in separate tab

**Review by wandercode — APPROVED** [review]

**Pull request merged**